impeller/entity/entity_pass.h

```diff
@@ -236,6 +236,7 @@ class EntityPass {
 
 struct CanvasStackEntry {
   Matrix xformation;
+  Rect clip_bounds;
```

Comment:
One question, does "coverage" imply exact coverage? This would be a conservative estimate of just the bounds of the device clip.

Reply:
I moved this struct to canvas.h since it's not used from anywhere else, and how do you feel about "cull_rect" for a name since it is only used for culling, not for clipping?

Reply:
Nah, "coverage" is always assumed to be a best effort minimum in Impeller. "cull_rect" SGTM.